src/components/AppProvider/utilities/Link/Link.ts

```diff
@@ -1,4 +1,4 @@
-import {LinkLikeComponent} from '../UnstyledLink';
```

Comment:
Keeps as a relative path for the moment

Reply:
(Gad this should be a comment on the diff line below, you changed this from a relative to absolute link)